src/clarity/deserialize.ts

```diff
@@ -0,0 +1,82 @@
+import { BufferReader } from '../utils';
+import { LengthPrefixedString, Address } from '..';
+import {
+  ClarityType,
+  ClarityValue,
+  intCV,
+  uintCV,
+  bufferCV,
+  trueCV,
+  falseCV,
+  standardPrincipalCVFromAddress,
+  contractPrincipalCVFromAddress,
+  responseOkCV,
+  responseErrorCV,
+  noneCV,
+  someCV,
+  listCV,
+  tupleCV,
+} from '.';
+
+export default function deserializeCV(bufferReader: BufferReader): ClarityValue {
```

Comment:
it would be useful if this also took a `Buffer` as an arg, e.g.
```ts
deserializeCV(input: BufferReader | Buffer) {
  const bufferReader = Buffer.isBuffer(input) ? new BufferReader(input) : input;
}
```